### ISSUE TITLE
Wrong source storage class displayed

### ISSUE THREAD

**Issue by sergiordlr** [fusor/mig-ui#623]
When migrating a "glusterfs-storage" volume, it is displayed as a "csi-rbd" volume in the migration screen.

This is what's displayed:
![image](https://user-images.githubusercontent.com/52459627/69556251-b7261180-0fa4-11ea-8a6c-752fb08ef63f.png)

And this is the actual PVC
```
]$ oc get pvc
NAME      STATUS    VOLUME                                     CAPACITY   ACCESSMODES   STORAGECLASS        AGE
mysql     Bound     pvc-35d1678e-0f85-11ea-8100-064d5e4c320a   1Gi        RWO           glusterfs-storage   2h
```

The same happens when trying to edit an existing migration plan. Source storage class is not identified correctly.

**Comment by ibolton336** [comment]
![bug?](https://user-images.githubusercontent.com/11218376/69573762-c0a77d80-0f94-11ea-9476-dc8ed6f8b209.gif)

**Comment by ibolton336** [comment]
Unable to reproduce

**Comment by sergiordlr** [comment]
The problem happens when we try to migrate from "glusterfs-storage".

When the source storage class is "glusterfs-storage" the storage class displayed is wrong. In the demo you linked, the source storage class used is gp2, not glusterfs.

**Comment by eriknelson** [comment]
This has been confirmed fixed by QE with the 1.1 release.
